ADMINS.md

```diff
@@ -1,38 +1,21 @@
 - [Overview](#overview)
 - [Current Admins](#current-admins)
 - [Admin Responsibilities](#admin-responsibilities)
-  - [Prioritize Security](#prioritize-security)
-  - [Enforce Code of Conduct](#enforce-code-of-conduct)
-  - [Adopt Organizational Best Practices](#adopt-organizational-best-practices)
 
 ## Overview
 
 This document explains who the admins are (see below), what they do in this repo, and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Current Admins
 
-| Admin                    | GitHub ID                                           | Affiliation |
-| -------------------------| ----------------------------------------------------| ----------- |
-| Charlotte                | [CEHENKLE](https://github.com/CEHENKLE)             | Amazon      |
-| Henri Yandell            | [hyandell](https://github.com/hyandell)             | Amazon      |
-| Rushi Agrawal            | [rushiagr](https://github.com/rushiagr)             | Amazon      |
-| Shephali Mittal          | [Shephalimittal](https://github.com/Shephalimittal) | Amazon      |
+| Admin           | GitHub ID                                           | Affiliation |
+| --------------- | --------------------------------------------------- | ----------- |
+| Charlotte       | [CEHENKLE](https://github.com/CEHENKLE)             | Amazon      |
```

Comment:
Charlotte is Charlotte Henkle.

I don't think anyone other than her and Henri are actual admins.